src/core/core_utils.js

```diff
@@ -611,6 +611,19 @@ function getRotationMatrix(rotation, width, height) {
   }
 }
 
+/**
+ * Get the number of bytes to use to represent the given positive integer.
+ * If n is zero, the function returns 0 which means that we don't need to waste
+ * a byte to represent it.
+ * @param {number} x - a positive integer.
+ * @returns {number}
+ */
+function getSizeInBytes(x) {
+  // n bits are required for numbers up 2^n - 1.
```

Comment:
Is this line perhaps missing a word?
```suggestion
  // n bits are required for numbers up to 2^n - 1.
```